src/main/resources/view/PersonListCard.fxml

```diff
@@ -32,7 +32,11 @@
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
-      <Label fx:id="instagram" styleClass="cell_small_label" text="\$instagram" />
+      <HBox alignment="CENTER_LEFT">
+        <ImageView fx:id="facebookIcon" fitHeight="14" fitWidth="14" pickOnBounds="true" preserveRatio="true" />
+        <Label fx:id="facebook" styleClass="cell_small_label" text="\$facebook" />
+        <Label fx:id="instagram" styleClass="cell_small_label" text="\$instagram" />
```

Comment:
I think you might be missing an `ImageView` for `instagram` here!

Reply:
Thanks for your patience! I've just pushed the commit which supports the Instagram icon.